src/proto/robot_config.proto

```diff
@@ -18,7 +18,110 @@ package bowler.robot_config;
 
 option java_multiple_files = true;
 option java_package = "com.commonwealthrobotics.proto.robot_config";
+//next 2
+// a center of mass object
+message CenterOfMass{
+  Pose centroid = 1;
+  double mass_kg = 2;
+}
 
-// Next: 1
+// next 3
+// a file location of a provider of some depency to be injected
+message GitFile {
+  string url = 1;
+  string branch = 2;
+  string file = 3;
+}
+// next 10
+message LimbType{
```

Comment:
I think you want this to be an enum rather than a message with a bunch of bools (assuming that the limb types are mutually exclusive?) See [here](https://github.com/CommonWealthRobotics/bowler-proto/blob/b11054084600b9e85a5ee5922d7cf06579752f78/src/proto/script_host.proto#L211-L222) for an example enum.

Related, will there be any additional data depending on the limb type? If so, then it should be a `oneOf` clause, not an enum or set of bools. Need to discuss this further in that case.

Reply:
no, Iron Mans arms have thrusters, Tachikoma's legs have wheels. they are not mutually exclusive